haskell/test/Test/FA2/BalanceOf.hs

```diff
@@ -27,7 +28,7 @@ test_unknown_position :: TestTree
 test_unknown_position =
   nettestScenarioOnEmulatorCaps "balance_of requires exising positions" do
     owner <- newAddress auto
-    cfmm <- fst <$> prepareSomeSegCFMM [owner]
+    cfmm <- fst <$> prepareSomeSegCFMM [owner] (FA2, CTEZ) def
```

Comment:
Probably it would be slightly better to use a dedicated `defTokenPair` constant for the case when we don't care which token pair is being tested - to remain consistent across test suites, but this is minor.

Reply:
Agreed, done.